style.css

```
@@ -495,7 +495,48 @@ body.dark .container .section .box {
 body.dark .container .inner-heading {
   color: #fff;
 }
+/* For Scroll Button*/
+#toTopBtn {
+  position: fixed;
+ bottom: 16px;
+ right: 29px;
+ z-index: 98;
+ padding: 21px;
+ background-color: #2666cf;
+ border-radius: 55%;
+}
 
+.js .cd-top--fade-out {
+ opacity: .5;
+}
+
+.js .cd-top--is-visible {
+ visibility: visible;
+ opacity: 1;
+}
+
+.js .cd-top {
+ visibility: hidden;
+ opacity: 0;
+ transition: opacity .3s,visibility .3s,background-color .3s;
+}
+
+.cd-top {
+ position: fixed;
+ bottom: 20px;
+ bottom: var(--cd-back-to-top-margin);
+ right: 20px;
+ right: var(--cd-back-to-top-margin);
+ display: inline-block;
+ height: 40px;
+ height: var(--cd-back-to-top-size);
+ width: 40px;
+ width: var(--cd-back-to-top-size);
+ box-shadow: 0 0 10px rgba(0,0,0,.05) !important;
```

```suggestion
 box-shadow: 0 0 10px rgba(0,0,0,.05) !important;
```
My suggestion was that using !important is not a good practice as it has the highest amount of specificity, so would you be able to make the box-shadow work without using !important.